### DISCUSSION TITLE
Bump version on files other than package.json

### DISCUSSION THREAD

**Discussion by elvis-pereira** [semantic-release/semantic-release#1812] [Q&A] [answered]
Hello, I started to use semantic release on a small project, for a npm package, and semantic release is just incredible! I have some other projects that I would like to start using semantic release too but they are not npm packages (although very similar), so I would need to bump the version on another json file, does semantic release allow that some way?

**Comment by gr2m** [comment, marked as answer]
Maybe this plugin could help? https://github.com/gr2m/semantic-release-plugin-update-version-in-files

**Reply by elvis-pereira** [reply]
If I correctly understood what it does, I don't think it will work. The version need to be written to the file and then back to the git repository using the git plugin, so each new version will end up changing the placeholder text. But this plugin at least gave me a direction, I my try to adapt it for what I need, thanks!

**Reply by gr2m** [reply]
I think you should be able to use https://github.com/gr2m/semantic-release-plugin-update-version-in-files in combination with the `git` plugin in order to push changes back to the repository.
